CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 Hydrophone is the module responsible for sending emails.
 This API sends notifications to users for things like forgotten passwords, initial signup, and invitations.
 
+## 1.12.2 - 2022-10-25
```

Comment:
Move than a fix, it should be 1.13 (yep we decide to be strict on the semver for go project)